scaffolder/publish-standalone-tech-docs/publish-standalone-tech-docs.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: publish-standalone-tech-docs
+  title: Publish a new standalone TechDocs repository and register with Roadie Backstage
+  description: "Publish a new repository with a catalog-info.yaml file with your company's standalone TechDocs details."
+spec:
+  owner: unspecified # TODO: Define who the owner of this template is. This is likely the team who own the relationship with Roadie. e.g. group:customer-name/platform
+  type: service
+
+  # These are the steps which are rendered in the frontend with the form input.
+  parameters:
+    - title: Repository Details # Title of the first step.
+      required:
+        - repoName
+      properties:
+        repoHost:
+          type: string
+          default: github.com
+          ui:widget: hidden
+
+        gitHubOrg:
+          type: string
+          default: roadie-demo # TODO: Set your GitHub Org value here.
+          ui:widget: hidden
+
+        repoName:
+          title: Repository Name
+          type: string
+          default: company-docs # TODO: Set your default name for the documentation repository here.
+
+        defaultBranch:
+          title: Repository Branch
+          type: string
+          default: main # TODO: Define your expected default branch.
+          enum: ['main']
+
+        repoVisibility:
+          title: Repository Visibility
+          type: string
+          default: private
+          enum: ['private', 'public']
+
+    - title: TechDocs Details # Title of the second step.
+      required:
+        - name
+        - description
+        - lifecycle
+      properties:
+        name:
+          title: Component Name
```

Comment:
Default this to the repository name.

Is there a way to add an explanation of why a Component Name is needed? It was surprising to me when I saw it, because I'm not creating a Component, I'm creating Documentation. When I thought about it, I realized that I need to have a Component so it shows up in the catalog, but that wasn't immediately obvious to me.